### ISSUE TITLE
Explicit handling of MatCheckbox clicks doesn't work with TalkBack on Android

### ISSUE THREAD

**Issue by kimberg** [angular/components#18005]
#### Reproduction

See:
https://stackblitz.com/edit/components-issue-emika8

Steps to reproduce:
1. Make a component that uses MAT_CHECKBOX_CLICK_ACTION noop to disable default checkbox handling, and an explicit click handler to toggle state.
2. Open in Chrome on Android, with TalkBack enabled.
3. Try to toggle the checkbox by double-tapping (normal selection gesture for TalkBack).

#### Expected Behavior

Checkbox can be toggled via TalkBack by double-tapping.

#### Actual Behavior

It cannot be toggled via TalkBack.

#### Environment

See:
https://stackblitz.com/edit/components-issue-emika8

- Angular: 
- CDK/Material:
- Browser(s): Chrome 79.0.3945.79
- Operating System (e.g. Windows, macOS, Ubuntu): Android 10

**Comment by jelbourn** [comment]
@crisbeto I'm wondering if this might be due to us letting the native action occur and then immediately setting the state back to what it was (rather than doing `preventDefault`)

**Comment by vick08** [comment]
@kimberg Do you happen to know whether the saem checkbox works with VoiceOver on iOS? How about other platforms?

**Comment by crisbeto** [comment]
We are closing this issue, because we've determined, using a semi-automated process, that it is no longer relevant, has low engagement, or it does not have enough information to proceed. Feel free to comment if you think the issue should be reopened, or open a new issue with the latest version of our code instead.
